plat/kvm/x86/time.c

```diff
@@ -38,6 +38,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <stdint.h>
```

Comment:
It makes sense that the change in the declaration (this header) must come together with updating the definition (in `plat/kvm/x86/time.c`). So perhaps you can also add a commit that updates the latter.

Also a general comment on your commit messages; there is no need for a line-break on every sentence. This is applicable only to documents. Instead, you should break at the 70-char limit similarly to:
```
Include `stdint/h` temporarily. Because of the removed include from
`include/uk/plat/time.h`, it is necessary to include it here. It will
be deleted when the libc data types from `plat/` are replaced with
Unikraft defined.
```